charts/dapr/crds/configuration.yaml

```diff
@@ -99,6 +99,8 @@ spec:
                           type: object
                         type:
                           type: string
+                        version:
```

Comment:
How would these new attributes be used?

Reply:
yes.  version field is used in httpPipeline.

```golang
// github.com/dapr/dapr/pkg/components/middleware/http/registry.go
func (p *httpMiddlewareRegistry) getMiddleware(name, version string) (func(middleware.Metadata) http_middleware.Middleware, bool) {
      nameLower := strings.ToLower(name)
      versionLower := strings.ToLower(version)
      middlewareFn, ok := p.middleware[nameLower+"/"+versionLower]
      if ok {
          return middlewareFn, true
      }
      if components.IsInitialVersion(versionLower) {
          middlewareFn, ok = p.middleware[nameLower]
      }
      return middlewareFn, ok
  }
```

Reply:
This method is kicked off for `kind: Middleware` CRDs, not for `kind: Configuration`.

Middlewares are defined as a pipeline in a Configuration object, but the Configuration object itself does not need a version field.

As such, I advise to close this PR. (but thanks for opening it nonetheless!)